src/components/timeline/ActiveTabGlobalTrack.js

```diff
@@ -72,10 +72,10 @@ class ActiveTabGlobalTrackComponent extends PureComponent<Props> {
     const { globalTrack } = this.props;
     switch (globalTrack.type) {
       case 'tab': {
-        const { mainThreadIndex } = globalTrack;
+        const { threadsKey } = globalTrack;
         return (
           <TimelineTrackThread
-            threadsKey={mainThreadIndex}
```

Comment:
I can't figure out how to get my own profile into active tab mode.

I would like to manually test active tab on this profile:

https://share.firefox.dev/3mc60Vz

I would assume this URL would get me into it:

https://deploy-preview-2752--perf-html.netlify.app/public/g2j0h638z94yy8e84ytn9chav9e1b5rpr38v870/calltree/?view=active-tab&v=5

Please leave this comment unresolved so that I can come back and handle the manual testing.